### ISSUE TITLE
deleteMessage isn't working - WAPI

### ISSUE THREAD

**Issue by dantenol** [open-wa/wa-automate-nodejs#1108]
**Describe the bug**
Messages don't get deleted on `deleteMessage()`

**Steps to Reproduce**
1. send a message to any number
2. try to delete that message using `wa.deleteMessage(chatId, [messageId], false)`
3. the function doesn't throw an error neither deletes the message

**create() code**
This is the code you use to create the client. e.g
```javascript
create(...)
```

**Expected behavior**
The message should be deleted for both sides

**DEBUG INFO**
```
{
  WA_VERSION: '2.2047.11',
  PAGE_UA: 'WhatsApp/2.2037.6 Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.83 Safari/537.36',
  WA_AUTOMATE_VERSION: '3.1.0',
  BROWSER_VERSION: 'Chrome/86.0.4240.198',
}
```

**Screenshots**
![Screenshot from 2020-11-23 16-13-09](https://user-images.githubusercontent.com/20271959/100004840-d8336500-2da6-11eb-8ca6-ef77c6625059.png)
![Screenshot from 2020-11-23 16-41-36](https://user-images.githubusercontent.com/20271959/100007393-c489fd80-2daa-11eb-9661-46e4d7b9a97a.png)

**Host (please complete the following information):**
 - OS: Ubuntu 20.04/android 10

**Additional context**
Apparently WAPI's Store.Msg doesn't capture all messages, For this reason, `getMessageById()` isn't working either.

**Comment by smashah** [comment]
@dantenol can you confirm that you're using `wa.deleteMessage` and not `wa.deleteMessages`. there is no `s` at the end of the method name

https://docs.openwa.dev/classes/client.html#deletemessage

**Comment by dantenol** [comment]
My bad. That's what I tried, as shown on screenshots. Just messed up on the issue. Already edited.

**Comment by zalviandyr** [comment]
Delete message doesnt work if you using whatsapp mod

**Comment by smashah** [comment]
@dante are you trying to delete a message that you've just sent or an old one?

**Comment by dantenol** [comment]
I always try to delete messages sent 2-3 minutes old. I'm aware of the time restrictions from WhatsApp.

What bothers me the most is that `getMessageById` isn't working either, and that's when the delete command goes wrong. Something is wrong on the WAPI side

**Comment by dantenol** [comment]
solved on 3.1.2

**Comment by smashah** [comment]
Great. Closing for now please report back if this issue happens again
